code/game/gamemodes/gameticker.dm

```diff
@@ -424,6 +429,32 @@ var/global/datum/controller/gameticker/ticker
 			return F
 
 
+/datum/controller/gameticker/proc/init_PDAgames_leaderboard()
+	init_snake_leaderboard()
+	init_minesweeper_leaderboard()
+
+/datum/controller/gameticker/proc/init_snake_leaderboard()
+	for(var/x=1;x<=9;x++)

```

Comment:
Why? there one ever be more or less speeds/labyrinths. Isn't the main point of defines being that you can easily change values?